com.playeveryware.eos/Documentation~/full_guide/01_starthere.md

```diff
@@ -0,0 +1,126 @@
+<a href="/README.md"><img src="/com.playeveryware.eos/Documentation~/images/PlayEveryWareLogo.gif" alt="README.md" width="5%"/></a>
+
+# Full Guide to Implementing the EOS Plugin for Unity
+
+This document serves as the starting point for a beginning-to-end guide on how to implement the EOS Plugin for Unity into your game.
+The only prerequisite to following this guide is having a game you want to implement the plugin into. This will walk you through
+the steps to set up your project, with links to documentation for more details and information.
+
+# Signing Up for Epic Online Services
+
+The EOS Plugin for Unity requires you to have a configured account and game on the Epic Game Store portal.
+For information about the benefits of publishing through Epic, see [Epic's Store Distribution portal](https://store.epicgames.com/en-US/distribution).
+This leads to [the Developer Portal landing page](https://dev.epicgames.com/portal/en-US/).
+Pressing Sign In will present you with a screen that you can sign up for a new account.
+If you already have an Epic Games Account, perhaps because you've played games on the Epic Games Store, you can sign in with that existing account.
+
+<a href="https://www.epicgames.com/id/login?"><img src="/com.playeveryware.eos/Documentation~/full_guide/epic_developerportal_signin_or_createaccount.png" alt="You can 'create account' on the Epic Developer Portal."/></a>
```

Comment:
It's meant to refer to the button labeled "Create Account". So the verb is "Create Account", rather than "create an account". I can change it as suggested though.